packages/app/src/components/ModeSelector.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div>
+    <div class="container centered-container" justify-around style="margin-top: 20px;">
```

Comment:
You've used flexbox well here and in your CSS, but I would replace these custom css classes with Quasar's inbuilt classes from https://quasar.dev/layout/grid/introduction-to-flexbox